lib/msal-core/package.json

```diff
@@ -29,49 +29,59 @@
     "doc": "typedoc --out ./docs ./src/ --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
     "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm test",
-    "test": "karma start && jasmine tests/MSALSpec.js",
+    "jtest": "karma start && jasmine-node --nohelpers tests/MSALSpec.js",
     "lint": "tslint -c tslint.json src/**/*.ts",
-    "build:webpack": "webpack"
+    "build:webpack": "webpack",
+    "test": "karma start --single-run --browsers PhantomJS karma.conf.js"
   },
   "dependencies": {
     "tslib": "1.7.1"
   },
   "devDependencies": {
+    "@types/chai": "^4.1.7",
     "@types/handlebars": "4.0.33",
-    "@types/jasmine": "2.5.41",
-    "@types/jasmine-ajax": "^3.1.39",
+    "@types/mocha": "^5.2.6",
+    "@types/node": "^11.11.5",
+    "@types/sinon": "^7.0.11",
     "ajv": "6.6.1",
     "atob": "~2.1.1",
     "awesome-typescript-loader": "^3.2.3",
     "babel-polyfill": "^6.26.0",
+    "chai": "^4.2.0",
     "fork-ts-checker-webpack-plugin": "^0.5.1",
     "istanbul-instrumenter-loader": "^3.0.1",
-    "jasmine": "^3.3.1",
-    "jasmine-ajax": "^3.3.1",
-    "jasmine-core": "^2.5.2",
+    "jsdom": "14.0.0",
+    "jsdom-global": "3.0.2",
     "jshint": "^2.9.7",
-    "karma": "^4.0.0",
-    "karma-chrome-launcher": "^2.0.0",
+    "karma": "^2.0.5",
+    "karma-chai": "^0.1.0",
+    "karma-chrome-launcher": "^2.2.0",
     "karma-cli": "^1.0.1",
-    "karma-coverage-istanbul-reporter": "^2.0.5",
-    "karma-firefox-launcher": "~0.1",
+    "karma-coverage-istanbul-reporter": "^2.0.1",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-html-reporter": "^0.2.7",
     "karma-jasmine": "^1.1.0",
     "karma-jasmine-ajax": "^0.1.13",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
     "karma-phantomjs-launcher": "*",
     "karma-remap-coverage": "^0.1.5",
     "karma-remap-istanbul": "^0.6.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-spec-reporter": "0.0.32",
-    "karma-typescript": "^4.0.0",
+    "karma-typescript": "^3.0.13",
     "karma-verbose-reporter": "0.0.6",
     "karma-webpack": "^4.0.0-rc.5",
+    "mocha": "^6.0.2",
     "phantomjs-polyfill": "0.0.2",
     "shx": "^0.2.2",
+    "sinon": "^7.3.1",
     "sourcemap-istanbul-instrumenter-loader": "^0.2.0",
     "ts-loader": "^5.3.1",
+    "ts-node": "^8.0.3",
     "tslint": "^5.8.0",
     "typedoc": "^0.14.2",
-    "typescript": "^2.5.3",
+    "typescript": "^3.2.1",
```

Comment:
This is a big bump on the version here. Any implications?

Reply:
I checked breaking changes here:
https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes

I have gone through the changes from 2.5 through 3.3, there aren't any obvious breaks which will affect our code directly. There may be some work for users to update usage, but those breaking changes will coincide with changes to the MSAL.js API surface.

This is required for using the latest mocha and chai versions.